circlecheck.lic

```diff
@@ -542,7 +542,8 @@ def display_requirements(requirements, level_target, brief, pretty_print)
     # This outputs the beginning section of circle check, which
     # gives your current circle and your TDPs at next level.
     DRC.message("___________________________________________________________")
-    respond("CURRENT CIRCLE:" + "#{DRStats.circle}" + " | Next level TDPS: " + "#{tdps}")
+    _respond "            Current Circle:  <pushBold/>#{DRStats.circle}<popBold/>" + 
```

Comment:
❓ What led to using `_respond` instead of `respond`? Line 545 and 547 use different method calls.

Reply:
The honest answer is that I wanted to use `<pushBold/>` on a portion of the line's text and it didn't seem to work with `respond` vs. `_respond`. Happy to learn anything additional, though.

Reply:
Example: pushbold used to highlight current circle and highlight the amount of next level's TDPs to visually separate it from the current TDPs.

![image](https://user-images.githubusercontent.com/6467969/147400610-f5184280-69b8-4600-9087-8973052286d1.png)

Reply:
Oh neat!